src/gf-platformid

```diff
@@ -41,7 +41,7 @@ coreos_gf_run_mount "${tmp_dest}"
 # * grub config
 # * BLS config (for subsequent config regeneration)
 # First, the grub config.
-if [ "$arch" != "x86_64" ]; then
+if [ "$arch" != "x86_64" ] && [ "$arch" != "s390x" ]; then
```

Comment:
It would be great if we explained in the comment above why we don't need to do this on these two platforms

Reply:
actually arm should be in this list too since we landed anaconda-less arm.

Reply:
@ajeddeloh should I add arm too? I don't have the possibility to test it on arm

Reply:
yeah

Reply:
@ajeddeloh  so I guess, I can simply state if `arch == ppc64le`. Is it still required for power because they use anaconda?

Reply:
Right. On non-anaconda the grub config uses BLS and thus doesn't have the entries in the grub config, which is what this path is handling. The anaconda path is doing a more traditional grub2-mkconfig.

`$arch == ppc64le` sgtm.

Reply:
@ajeddeloh I updated the PR